[2024-03-22T10:13:23.155+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_to_bigquery.create_table manual__2024-03-22T10:13:16.480329+00:00 [queued]>
[2024-03-22T10:13:23.171+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_to_bigquery.create_table manual__2024-03-22T10:13:16.480329+00:00 [queued]>
[2024-03-22T10:13:23.172+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-03-22T10:13:23.193+0000] {taskinstance.py:2191} INFO - Executing <Task(BigQueryCreateEmptyTableOperator): create_table> on 2024-03-22 10:13:16.480329+00:00
[2024-03-22T10:13:23.201+0000] {standard_task_runner.py:60} INFO - Started process 1274 to run task
[2024-03-22T10:13:23.205+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl_to_bigquery', 'create_table', 'manual__2024-03-22T10:13:16.480329+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp0v3wg4r3']
[2024-03-22T10:13:23.208+0000] {standard_task_runner.py:88} INFO - Job 10: Subtask create_table
[2024-03-22T10:13:23.266+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_to_bigquery.create_table manual__2024-03-22T10:13:16.480329+00:00 [running]> on host 8a685a05c3f5
[2024-03-22T10:13:23.375+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='minhduc' AIRFLOW_CTX_DAG_ID='etl_to_bigquery' AIRFLOW_CTX_TASK_ID='create_table' AIRFLOW_CTX_EXECUTION_DATE='2024-03-22T10:13:16.480329+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-22T10:13:16.480329+00:00'
[2024-03-22T10:13:23.389+0000] {connection.py:234} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-03-22T10:13:23.392+0000] {base.py:83} INFO - Using connection ID 'gcp' for task execution.
[2024-03-22T10:13:23.393+0000] {bigquery.py:1412} INFO - Creating table
[2024-03-22T10:13:25.422+0000] {bigquery.py:1440} INFO - Table retail.raw_invoices already exists.
[2024-03-22T10:13:25.463+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl_to_bigquery, task_id=create_table, execution_date=20240322T101316, start_date=20240322T101323, end_date=20240322T101325
[2024-03-22T10:13:25.508+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-22T10:13:25.535+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
